extensions/wikia/PhalanxII/tests/PhalanxModelTest.php

```diff
@@ -224,189 +224,181 @@ public function phalanxNewFromTypeProvider() {
 	}
 
 	/* data providers */
-	public function phalanxUserModelDataProvider() {
+	public function phalanxUserModelDataProvider(): array {
 		/* valid user */
-		$validUser = array(
-			'isAnon'    	=> false,
-			'getName'   	=> self::VALID_USERNAME,
-			'email'			=> self::VALID_EMAIL,
-			'block'     	=> 0,
-			'result'    	=> 1,
-			'error'			=> ''
-		);
+		$validUser = [
+			'isAnon' => false,
+			'getName' => self::VALID_USERNAME,
+			'email' => self::VALID_EMAIL,
+			'block' => 0,
+			'result' => 1,
+			'error' => ''
+		];
 
 		/* invalid user */
-		$invalidUser = array(
-			'isAnon'    	=> true,
-			'getName'   	=> self::INVALID_USERNAME,
-			'email'			=> self::INVALID_EMAIL,
-			'block'     	=> (object) array(
+		$invalidUser = [
+			'isAnon' => true,
+			'getName' => self::INVALID_USERNAME,
+			'email' => self::INVALID_EMAIL,
+			'block' => (object) [
 				'regex' => 0,
 				'expires' => '',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "expires" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)